void main(List<String> args) {
  // runtime error and Complie time error
   // exception handling
  // fetching,
    
   try {
    int res = 10~/0; 
   } catch (e) {
    print(e);
   } finally {
    print("3 This is the finally block");
   }
   
  print("13 End of the program");
}